packages/swapper/src/swappers/ThorchainSwapper/utils/longTailHelpers.ts

```diff
@@ -45,10 +45,10 @@ export function getTradeType(
 ): TradeType | undefined {
   switch (true) {
     case !!sellAssetPool && !!buyAssetPool:
-    case !!buyAssetPool && !sellAssetPool && sellPoolId === 'THOR.RUNE':
-    case !!sellAssetPool && !buyAssetPool && buyPoolId === 'THOR.RUNE':
+    case !!buyAssetPool && !sellAssetPool && ['THOR.RUNE', 'THOR.TCY'].includes(sellPoolId ?? ''):
+    case !!sellAssetPool && !buyAssetPool && ['THOR.RUNE', 'THOR.TCY'].includes(buyPoolId ?? ''):
       return TradeType.L1ToL1
-    case sellPoolId === 'THOR.RUNE' && !buyAssetPool:
+    case ['THOR.RUNE', 'THOR.TCY'].includes(sellPoolId ?? '') && !buyAssetPool:
```

Comment:
Just confirming tcy is supported for l1 to longtail? I don't see why it wouldn't be, but checking if you know this for a fact or have tested it.

Reply:
Absolutely is, it's just like any other pool!
<img width="1061" alt="image" src="https://github.com/user-attachments/assets/68fd53f6-28f5-4d0f-a7c3-bbd28970559b" />
<img width="1046" alt="image" src="https://github.com/user-attachments/assets/2c6051cd-8a1b-4076-bcfc-2c5e689acccd" />